### PR TITLE
Remove captcha on examples

### PR DESCRIPTION
**Pull request by ryanannatuvasquez**
## Description
- Remove captcha examples, it will be no longer need when rejecting a job

### CODE REVIEW

File: examples/updateTranslationJob.py [hunk 1]
```diff
@@ -44,8 +44,8 @@
     debug=True
 )
 
-# Update a job that has an id of 42, and reject it, cite the reason,
-# add a comment, and throw up some captcha stuff. See the docs for
+# Update a job that has an id of 42, and reject it, cite the reason
+# and add a comment. See the docs for
```
Comment (jeanklingler) [inline examples/updateTranslationJob.py]: you forgot that line https://github.com/gengo/gengo-python/blob/7364afdff49accf82988f4176fdc4730d614e942/examples/updateTranslationJob.py#L54

**Comment by ryanannatuvasquez** [comment]
@jeanklingler @josephniel @narusemotoki PTAL

**Comment by jeanklingler** [comment]
you should probably rebase and double check the CI :slightly_smiling_face:

**Comment by ryanannatuvasquez** [comment]
@jeanklingler sure I will rebase, however the error in CI only occurring in python3.3 
```
"pyflakes" failed during execution due to "'str' object has no attribute 'arg'"
Run flake8 with greater verbosity to see more details
```

File: examples/updateTranslationJob.py (hunk 1 above)
Comment (ryanannatuvasquez) [reply examples/updateTranslationJob.py]: Fixed in https://github.com/gengo/gengo-python/pull/103/commits/05393b2063119433887ab4966275816cb8c3baba

**Comment by ryanannatuvasquez** [comment]
@narusemotoki im not sure if its necessary, we dont change any implementation of behavior. maybe its not needed for this change.

**Comment by narusemotoki** [comment]
But otherwise we can't upload to PyPI because version conflicting.

**Pull request merged**